package proto_ff;

import "yd_fieldoptions.proto";


message achievementitemRewardDesc
{
	optional int64 ID = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 Value = 2[(yd_fieldoptions.field_cname) = "数量"];
}

message achievementachievement
{
	optional int64 achievementId = 1[(yd_fieldoptions.field_cname) = "成就ID"];
	optional int32 achievementLevel = 2[(yd_fieldoptions.field_cname) = "成就触发等级"];
	optional int32 achievementDTP = 3[(yd_fieldoptions.field_cname) = "成就大类"];
	optional int32 achievementType = 4[(yd_fieldoptions.field_cname) = "成就类型"];
	optional int32 achievementSubType = 5[(yd_fieldoptions.field_cname) = "事件类型"];
	optional int64 achievementObjectID = 6[(yd_fieldoptions.field_cname) = "事件目标ID"];
	optional string achievementObjectIDString = 7[(yd_fieldoptions.field_cname) = "事件目标Idstring", (yd_fieldoptions.field_bufsize) = 32];
	optional int32 achievementObjectValue = 8[(yd_fieldoptions.field_cname) = "事件目标完成值"];
	optional int32 achievementDia = 9[(yd_fieldoptions.field_cname) = "获得绑定钻石数量"];
	optional int32 achievementGod = 10[(yd_fieldoptions.field_cname) = "获得金币数量"];
	optional int32 achievementPoint = 11[(yd_fieldoptions.field_cname) = "获取成就点数量"];
	optional int32 linkID = 12[(yd_fieldoptions.field_cname) = "前往ID"];
	repeated achievementitemRewardDesc itemReward = 13[(yd_fieldoptions.field_cname) = "获得物品", (yd_fieldoptions.field_arysize) = 2];
}


message Sheet_achievementachievement
{
	repeated achievementachievement achievementachievement_List = 1[(yd_fieldoptions.field_arysize)=429];
}

message achievementachievementreward
{
	optional int64 achievementPrizeId = 1[(yd_fieldoptions.field_cname) = "成就奖励ID"];
	optional int32 achievementType = 2[(yd_fieldoptions.field_cname) = "成就类型"];
	optional int32 achievementPointValue = 3[(yd_fieldoptions.field_cname) = "所需成就点值"];
	optional int64 achievementSkil = 4[(yd_fieldoptions.field_cname) = "成就技能id"];
	optional int32 achievementDia = 5[(yd_fieldoptions.field_cname) = "获得绑定钻石数量"];
}


message Sheet_achievementachievementreward
{
	repeated achievementachievementreward achievementachievementreward_List = 1[(yd_fieldoptions.field_arysize)=110];
}

message achievementachievenName
{
}


message Sheet_achievementachievenName
{
	repeated achievementachievenName achievementachievenName_List = 1[(yd_fieldoptions.field_arysize)=119];
}
